app/uploaders/avatar_uploader.rb

```diff
@@ -0,0 +1,47 @@
+# encoding: utf-8
+class AvatarUploader < CarrierWave::Uploader::Base
+  # Include RMagick or MiniMagick support:
+  # include CarrierWave::RMagick
+  # include CarrierWave::MiniMagick
+
+  # Choose what kind of storage to use for this uploader:
+  storage :file
```

Comment:
you will need to support Amazon S3 for this to be a deployable feature. We'll cover this next week, but there is also apollo curriculum for supporting this.